inductiva/molecules/scenarios/utils.py

```diff
@@ -14,3 +15,21 @@ def unwrap_trajectory(topology, trajectory):
     transformation = transformations.unwrap(atoms)
     universe.trajectory.add_transformations(transformation)
     return universe
+
+
+def align_trajectory_to_average(universe, trajectory_output_path):
+    """Align the trajectory to the average structure. 
+    Args:
+        universe: The universe MDAnalysis object.
+        trajectory_output_path: Path to the aligned trajectory file."""
+    average = align.AverageStructure(universe,
+                                     universe,
+                                     select="protein and name CA",
+                                     ref_frame=0).run()
+    average_trajectory = average.results.universe
+
+    align.AlignTraj(universe,
+                    average_trajectory,
+                    select="protein and name CA",
+                    filename=trajectory_output_path,
+                    in_memory=False).run()
```

Comment:
Can you elaborate in what does this do?

Reply:
First we compute an average structure with the average coordinates of each atom. Then we align the trajectory to that  reference. We can look into more details [here](https://userguide.mdanalysis.org/stable/examples/analysis/alignment_and_rms/rmsf.html).

Reply:
Add a comment with that link here.